### PR TITLE
3.0 Upgrade Guide: Added Table of Contents

### PR DESCRIPTION
**Pull request by smithandre**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR | NA |

Added a table of contents, which helps me find relevant upgrade information a bit faster.  It's very basic and straight forward, as I don't want to add too much overhead in terms of maintaining these guides.

### CODE REVIEW

File: UPGRADE-3.0.md [hunk 1]
```diff
@@ -381,6 +410,7 @@ UPGRADE FROM 2.x to 3.0
    $form = $this->createForm(MyType::class);
    ```
 
+ *
```
Comment (javiereguiluz) [inline UPGRADE-3.0.md]: Is this empty element needed?

**Review by javiereguiluz — APPROVED** [review]
👍 

Although this will make the maintenance of this file a bit harder, I think this is very useful.

File: UPGRADE-3.0.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+- [Translator](#translator)
+- [Twig Bridge](#twig-bridge)
+- [TwigBundle](#twigbundle)
+- [Validator](#validator)
+- [Yaml](#yaml)
+- [WebProfiler](#webprofiler)
+- [Process](#process)
+- [Monolog Bridge](#monolog-bridge)
+- [Swiftmailer Bridge](#swiftmailer-bridge)
+- [HttpFoundation](#httpfoundation)
```
Comment (xabbuh) [inline UPGRADE-3.0.md]: imo this should be sorted alphabetically to make it easier to find what you are looking for

Comment (smithandre) [inline UPGRADE-3.0.md]: Initially I added it in the order that one finds the sections on the page. It does however make sense to list it alphabetically in the Table of contents

File: UPGRADE-3.0.md (hunk 1 above)
Comment (smithandre) [inline UPGRADE-3.0.md]: Nope. Removed it

**Comment by fabpot** [comment]
Thank you @smithandre.

**Pull request closed without merging**